.github/CODEOWNERS

```diff
@@ -414,6 +414,8 @@
 /core/templates/services/interaction-specs.service*.ts @bansalnitish
 /core/templates/services/translation-file-hash-loader.service.ts @nithusha21
 /release_constants.py @ankita240796 @nithusha21 @DubeySandeep
+/core/domain/image_service*.py @seanlip
+
```

Comment:
Don't add the extra blank line. Keep two blank lines between sections. (In general, check the "Files Changed" tab in GitHub and make sure the changes are exactly what you want them to be.)